webrtc.html

```diff
@@ -7295,6 +7295,7 @@ <h2>Methods</h2>
     required DOMHighResTimeStamp timestamp;
     required unsigned long       source;
              double              audioLevel;
+             unsigned long       rtpTimestamp;
```

Comment:
Note that *RTCRtpSynchronizationSource* is derived from *RTCRtpContributingSource*. Did you mean to add *rtpTimestamp* to both? Just making sure it's not done by accident.

Reply:
I don't know the details of this well enough to tell which is the correct place to put it. To me it makes  sense put it where you have the other timestamp in contributing source though.

Reply:
I think it belongs here. Both the SSRC and CSRC dictionary contains information from a particular RTP packet. This is more information about that RTP packet, so it can be added to both. (For the intended use case it would be enough to have it in the SSRC dictionary, but it doesn't hurt to have it here too.)